runtime/interpreter/interpreter.go

```diff
@@ -4991,7 +4991,7 @@ func (interpreter *Interpreter) mapMemberValueAuthorization(
 
 		default:
 			var access sema.Access
-			if mappedAccess.Type == sema.IdentityType {
+			if mappedAccess.Type.IncludesIdentity {
```

Comment:
Could you please add a comment to `IncludesIdentity` and explain for when it is set and how it is used?